src/System.Security.Permissions/src/System.Security.Permissions.csproj

```diff
@@ -172,6 +169,11 @@
     <Compile Include="System\Transactions\DistributedTransactionPermission.cs" />
     <Compile Include="System\Transactions\DistributedTransactionPermissionAttribute.cs" />
   </ItemGroup>
+  <ItemGroup Condition="'$(TargetGroup)' == 'netcoreapp' or '$(TargetGroup)' == 'uap'">
```

Comment:
These types should be included in the netstandard version only. For uap and netcoreapp, they are in System.Runtime.Extensions with this change.

Reply:
Thanks will update